javaScript-week1/Ez-Namey.js

```diff
@@ -0,0 +1,31 @@
+let firstWords = [
+    "Alpha",
+    "Bravo",
+    "Charlie",
+    "Delta",
+    "Echo",
+    "Foxtrot",
+    "Golf",
+    "Hotel",
+    "India",
+    "Juliet",
+];
+let secondWords = [
+    "Kilo",
+    "Lima",
+    "Mike",
+    "November",
+    "Oscar",
+    "Papa",
+    "Quebec",
+    "Romeo",
+    "Sierra",
+    "Tango",
+];
+
+const randomNumber1 = Math.floor(Math.random() * 10); 
+const randomNumber2 = Math.floor(Math.random() * 10); 
+const startupName = firstWords[randomNumber1] + " " + secondWords[randomNumber2];
+console.log(
+  `"The startup: ${startupName}, contains ${startupName.length} characters`
```

Comment:
There is a quote mark you can remove right before `The`

Reply:
i am on it